docker-compose.yml

```diff
@@ -5,16 +5,33 @@ services:
       args:
         tags: netgo osusergo sqlite_foreign_keys sqlite_math_functions sqlite_omit_load_extension sqlite_unlock_notify sqlite_vacuum_incr
     depends_on:
+      - minio
       - postgres
     environment:
-      FML_DATABASE_URI: postgres://postgres:postgres@postgres/postgres
       FML_LOG_LEVEL: debug
+      FML_DATABASE_URI: postgres://postgres:postgres@postgres/postgres
+      FML_S3_ENDPOINT_URL: http://minio:9000
+      AWS_ACCESS_KEY_ID: "YFptmVrlVdOgnHZ8Idse"
+      AWS_SECRET_ACCESS_KEY: "ZeKsyUJikIAJQVtYsQNtdXx8qv9eevL94qFOx0Jh"
```

Comment:
why are you using different credentials here compared to the devcontainer?

Reply:
made them the same.